client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -38,6 +42,15 @@ class DomainRegistrationSuggestion extends React.Component {
 			domain_name: PropTypes.string.isRequired,
 			product_slug: PropTypes.string,
 			cost: PropTypes.string,
+			match_reasons: PropTypes.arrayOf(
+				PropTypes.oneOf( [
+					'tld-exact',
```

Comment:
I know we use other strings all over the place, but it might be nice to make these constants in `lib/domains/constants.js` to help catch typos since they're used in multiple files.